spec/system/avo/tags_spec.rb

```diff
@@ -143,8 +143,12 @@
       wait_for_tags_to_load(field_value_slot)
       type(:down, :return)
 
-      sleep 0.3
-      click_on "Run"
+      # TODO: fix "found unpermitted parameter: :user_id-dummy"
+      # tags field passes a dummy param generated from a fake input which is always unpermitted
+      with_temporary_class_option(ActionController::Parameters, :action_on_unpermitted_parameters, :log) do
+        sleep 0.3
+        click_on "Run"
+      end
```

Comment:
Using a tags field will always raise the unpermitted error since it is generating a dummy param that isn't permitted anywhere.